pkgs/by-name/sc/screenconfig/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  xrandr,
+  srandrd,
+  feh,
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

Comment:
```suggestion
python3.pkgs.buildPythonApplication rec {
```

Reply:
Still a problem — `buildPythonPackage` should only be used if a package is in the Python package set under `pkgs/development/python-module`, and `buildPythonApplication` should be used otherwise